### PR TITLE
Create Other Category of Troubleshooting Kubernetes Integration Issues

### PR DESCRIPTION
**Pull request by isaacadeleke-nr**
There are some known issues that we want our customers to know how to solve while we work on fixing them so they are not frustrated during their usage so we add an other category of troubleshooting help.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/other.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: 'Kubernetes integration troubleshooting: Other Problems'
+type: troubleshooting
+tags:
+  - Integrations
+  - Kubernetes integration
+  - Troubleshooting for other category of problems you might face in your New Relic Kubernetes integration.
+---
+
+## Known Issues
```
Comment (ally-sassman) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/other.mdx:10]:
```suggestion
## Known issues
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/other.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  - Integrations
+  - Kubernetes integration
+  - Troubleshooting for other category of problems you might face in your New Relic Kubernetes integration.
+---
+
+## Known Issues 
+
+### Problem 
+
+You have completed the [installation procedure](/docs/kubernetes-monitoring-integration#install) for New Relic's Kubernetes integration with helm for the nri-bundle, but the values in your values.yaml file are not being respected
```
Comment (ally-sassman) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/other.mdx:14]:
```suggestion
You have completed the [installation procedure](/docs/kubernetes-monitoring-integration#install) for New Relic's Kubernetes integration with helm for the `nri-bundle`, but the values in your `values.yaml` file are not being respected.
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/other.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+### Solution
+
+Some [global values](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle#values) have been found to not be respected by our helm templates, so if you find that a global value isn't being respected by the appropriate chart, directly specify the chart. This is a workaround and the issue will be fixed.
```
Comment (ally-sassman) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/other.mdx:18]:
```suggestion
Some [global values](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle#values) have been found to not be respected by our helm templates, so if you find that a global value isn't being respected by the appropriate chart, directly specify the chart. This workaround should fix the issue. 
```

**Comment by isaacadeleke-nr** [comment]
Hey @ally-sassman since a new file was created, should there be some sort of routing update to a routing file? I saw mention that it would be taken care of if I wasn't sure how to do it myself.

**Comment by ally-sassman** [comment]
@isaacadeleke-nr Ah yes, you're right and good catch! I need to add it to one of our navigation files to appear in the left nav. I'll open a new PR and tag you in it soon.

**Pull request merged**